template/README.md

```diff
@@ -182,20 +203,39 @@ vagrant ssh -c 'vault read consul/creds/dev-team'
 vagrant ssh -c 'vault read consul/creds/admin-team'
 ```
 
-*Tokens can be used to access UI (different access level depends on role)
+> :bulb: Tokens can be used to access UI (different access level depends on role)
+
+
+## Usage
+### Commands
+There are several commands that help to run the vagrant-box:
+- `make install` installs all prerequisites
+
+- `make up` provisions a [vagrant-hashistack](https://github.com/fredrikhgrelland/vagrant-hashistack/) box on your machine. After the machine and hashistack are set up it will run the [Default Startup Scheme](#default-startup-scheme).
+
+- `make clean` takes down the provisioned box if there is any.
+
+- `make update` downloads the newest version of the [vagrant-hashistack box](https://github.com/fredrikhgrelland/vagrant-hashistack/) from [vagrantcloud](https://vagrantcloud.com/fredrikhgrelland/hashistack).
+
+- `make example` runs the example in [template_example/](template_example)
+
```

Comment:
```suggestion
`make dev` - the same as `make up`, but skip all tests.
```

Reply:
This is not true. Skips only https://github.com/fredrikhgrelland/vagrant-hashistack/blob/dcab1d24280e28bf2639a2b16545e73069ce83f3/ansible/bootstrap.yml#L11-L15 atm

Reply:
What the differences then or point to have `up` and `dev`?

Reply:
Right now there is no point (which is why I created issue #243).

Reply:
Ok. we will fix it in another PR